easybuild/easyconfigs/p/PICRUSt2/PICRUSt2-2.5.2-foss-2022b.eb

```diff
@@ -32,8 +32,18 @@ exts_list = [
     }),
 ]
 
+postinstallcmds = [
+    "cp -a %(start_dir)s%(name)s/%(namelower)s-%(version)s/%(namelower)s/default_files %(installdir)s/lib/python%(pyshortver)s/site-packages/%(namelower)s/"
```

Comment:
You can solve this by using implicit string concationation. I.e.

``` 
postinstallcmds = [
   "cp -a something "
   "something_else"
]
```
gets automatically concatenated by Python to

``` 
postinstallcmds = [
   "cp -a something something_else"
]
```
because it is between brackets.

I think that'll enable you to keep this below the line limit when breaking after `default_files ` (i.e. after the space), which is probably the most 'pretty' place to break.